docs/src/hdf5.md

```diff
@@ -73,16 +73,16 @@ Flat ``n``-dimensional arrays are stored as ``n``-dimensional HDF5 datasets.
 
 A vector of vectors of unqual sizes is stored as an HDF5 group that contains two datasets:
 
-* A 1-dimensional dataset `flattened_data` that stores the concatenation of all vectors into a single vector.
-* A 1-dimensional dataset `cumulative_length` that stores the cumulative sum of the length of all vectors.
+* An array-like dataset `flattened_data` that stores the concatenation of all vectors into a single vector. Can be `*array<n>{...}`, `table{...}`, etc.
```

Comment:
The flattened data should always be a one-dimensional vector, I think. If we want to support vectors of multi-dimensional arrays of non-equal size, we need an additional dataset that provides the size of dim 2 to n of each member arrays (in Julia, in `ArraysOfArrays.VectorOfArrays` we use an additional vector `kernel_size` for this).